README.md

```diff
@@ -6,21 +6,23 @@
 
 MOJ Frontend contains the code you need to start building user interfaces for UK Ministry of Justice government services.
 
-See live examples of MOJ Frontend components, and guidance on when to use them in your service, in the [MOJ Design System](https://moj-design-system.herokuapp.com/).
+See live examples of MOJ Frontend components, and guidance on when to use them in your service, in the [MOJ Design System](https://design-patterns.service.justice.gov.uk/).
 
 ## Contribution Guidelines
 
 If you want to help us build MOJ Frontend, view our [contribution guidelines](CONTRIBUTING.md). This covers all areas from semvar commit messages to release process.
 
 ## Contact the team
 
-MOJ Frontend is maintained by the MOJ Design System team. If you need support [contact the Design System team](https://moj-design-system.herokuapp.com/get-in-touch).
+MOJ Frontend is maintained by staff in the Ministry of Justice. If you need support, you can use [GitHub discussions](https://github.com/ministryofjustice/moj-frontend/discussions) or one of our Slack channels:
+- [#moj-design-system-support](https://mojdt.slack.com/archives/CH5RUSB27) on MOJ Digital & Technologyfix-
```

Comment:
Is the word `fix` meant to be at the end of this line or is it a typo?

Reply:
Very much a typo! Fixed now :)